java-lib-test/settings.gradle.kts

```diff
@@ -0,0 +1,10 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/8.1.1/userguide/multi_project_builds.html
+ */
+
+rootProject.name = "java-lib-test"
```

Comment:
Should this be `example-client` or something like that?

Reply:
calling it `example-graphsdk-client`